pubsub/integration_test.go

```diff
@@ -1217,8 +1217,7 @@ func TestIntegration_OrderedKeys_JSON(t *testing.T) {
 	scanner := bufio.NewScanner(inFile)
 	for scanner.Scan() {
 		line := scanner.Text()
-		// TODO: use strings.ReplaceAll once we only support 1.11+.
-		line = strings.Replace(line, "\"", "", -1)
+		line = strings.ReplaceAll(line, "\"", "")
```

Comment:
This was adding in 1.12. Need to keep using Replace for now to stay compatible.

Reply:
You're right, should've checked beforehand instead of relying on the comment. I'll change this PR to only remove the experimental notice.